geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/ProtobufSerializationService.java

```diff
@@ -120,7 +122,7 @@ public Object decode(BasicTypes.EncodedValue encodedValue) throws EncodingExcept
         return encodedValue.getStringResult();
       case JSONOBJECTRESULT:
         return jsonPdxConverter.decode(encodedValue.getJsonObjectResult());
-      case VALUE_NOT_SET:
+      case NULLRESULT:
```

Comment:
Should we still handle `VALUE_NOT_SET`? Or has that gone the way of the dodo?

Reply:
Well, it's part of the protobuf spec, so we can't really get rid of it, but we're no longer going to support it.  An EncodedValue without a type set will be considered incorrectly encoded and throw an exception.